Library/Homebrew/test/support/helper/spec/shared_context/homebrew_cask.rb

```diff
@@ -39,15 +39,26 @@ class Config
       Cask::Config::DEFAULT_DIRS_PATHNAMES.each_value(&:mkpath)
 
       CoreCaskTap.instance.tap do |tap|
-        tap.cask_dir.mkpath
-        (TEST_FIXTURE_DIR/"cask/Casks").children.each do |casks_path|
-          FileUtils.ln_sf casks_path, tap.cask_dir
+        fixture_cask_dir = TEST_FIXTURE_DIR/"cask/Casks"
+        fixture_cask_dir.glob("**/*.rb").each do |fixture_cask_path|
+          relative_cask_path = fixture_cask_path.relative_path_from(fixture_cask_dir)
+
+          # These are only used manually in tests since they
+          # would otherwise conflict with other casks.
+          next if relative_cask_path.dirname.basename.to_s == "outdated"
```

Comment:
I wonder if we could just include this in the glob pattern itself. Doesn't need to be changed though. This is fine.